test/cql-pytest/test_bloom_filter.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+# Test inserts `N` rows into table, flushes it 
+# and tries to read `M` non-existing keys.
+# Then bloom filter's false-positive ratio is checked.
+
+import pytest
+import random
+import nodetool
+from util import new_test_table
+
+def test_bloom_filter(cql, test_keyspace):
+    N = 10000 # Number of writes
+    M = 5000  # Number of reads
+    fp_chance = 0.1
+
+    keys = random.sample(range(2147483647), N+M)
+    with new_test_table(cql, test_keyspace, "a int PRIMARY KEY", 
+        f"WITH bloom_filter_fp_chance = {fp_chance}") as table:
+        
+        stmt = cql.prepare(f"INSERT INTO {table} (a) VALUES(?)")
+        for k in keys[:N]:
+            cql.execute(stmt, [k])
+        nodetool.flush(cql, table)
+        
+        for k in keys[N:N+M]:
+            cql.execute(f"SELECT * FROM {table} WHERE a = {k} BYPASS CACHE")
+
+        fp = nodetool.bloom_filter_false_positives(cql, table)
```

Comment:
It's not enough to fetch this metric once - you need to run it once in the beginning of the test, once at the end, and subtract.
This is still problematic - it won't work if tests are running in parallel on the same Scylla - but better than what you have now that basically assumes that this is the first test to run on this Scylla.

Reply:
I'm using `with new_test_table(cql, test_keyspace, "a int PRIMARY KEY", f"WITH bloom_filter_fp_chance = {fp_chance}") as table:` so if i'm not mistaken, fetching metic more times isn't required, cause this is new table.

The problem is if the test is running on multiple node Scylla cluster. For me, it looks like nodetools/REST API returns metric from single node, not whole table, because if the test is running on `n`-nodes cluster, the ratio looks like devided by `n`. Is there any fixture to run this test on single node cluster?